Cargo.lock

```diff
@@ -7887,38 +7887,6 @@ version = "0.3.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "c08d65885ee38876c4f86fa503fb49d7b507c2b62552df7c70b2fce627e06381"
 
-[[package]]
-name = "opendal"
-version = "0.47.2"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "ff159a2da374ef2d64848a6547943cf1af7d2ceada5ae77be175e1389aa07ae3"
-dependencies = [
- "anyhow",
- "async-trait",
- "backon",
- "base64 0.22.0",
- "bytes",
- "chrono",
- "crc32c",
- "flagset",
- "futures",
- "getrandom",
- "http 1.1.0",
- "log",
- "md-5",
- "once_cell",
- "percent-encoding",
- "prometheus",
- "quick-xml 0.31.0",
- "reqsign 0.15.2",
- "reqwest 0.12.4",
- "serde",
- "serde_json",
- "sha2",
- "tokio",
- "uuid",
-]
-
 [[package]]
 name = "opendal"
 version = "0.49.0"
```

Comment:
Shall we upgrade to 0.49.2 instead of 0.49.0?

Reply:
OpenDAL just release v0.50 5days ago🥲.

Reply:
We don't need to rush to 0.50. Given that we specify `0.49` in `Carago.toml`, I think we should bump to the latest `0.49` which is `0.49.2`